src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/media/index.js

```diff
@@ -78,7 +99,25 @@ const withValidationCheck = convertedBlock => {
   return checks.every(Boolean) && convertedBlock;
 };
 
-const convertMedia = block => {
+const validateInputBlock = (block, aresResponse) => {
+  const log = category =>
+    logger.warn(category, {
+      id: path(['metadata', 'id'], aresResponse),
+      url: path(['metadata', 'locators', 'assetUri'], aresResponse),
+    });
+
+  [
+    [CPS_MEDIA_WITHOUT_IMAGE, block.imageUrl],
+    [CPS_MEDIA_WITHOUT_ID, block.id],
+    [CPS_MEDIA_WITHOUT_FORMAT, block.format],
+    [CPS_MEDIA_WITHOUT_VERSION_ID, path(['versions', 0, 'versionId'], block)],
+  ]
+    .filter(([, checkSucceeded]) => !checkSucceeded)
+    .forEach(([issue]) => log(issue));
```

Comment:
I'd suggest changing it so that we have a single event - `CPS_MEDIA_MISSING_FIELD`, but with the missingField being different. It'll make it easier to handle observability of issues related to CPS Media assets.
```suggestion
  const log = missingField =>
    logger.warn(CPS_MEDIA_MISSING_FIELD, {
      id: path(['metadata', 'id'], aresResponse),
      url: path(['metadata', 'locators', 'assetUri'], aresResponse),
      missingField,
    });

  [
    ['imageUrl, block.imageUrl],
    ['id', block.id],
    ['format', block.format],
    ['versionId', path(['versions', 0, 'versionId'], block)],
  ]
    .filter(([, checkSucceeded]) => !checkSucceeded)
    .forEach(([issue]) => log(issue));
```

Reply:
ah thanks @sareh - hadn't realised this was already done as a github suggestion

and I guess github support multiline suggestions now!